Automattic/jetpack
# Docker: Add psySH as default wp shell REPL

#### Changes proposed in this Pull Request:

* adds [psySH](https://psysh.org) REPL into WP image;
* use `psySH` as default REPL in `wp shell`

#### Testing instructions:






* Delete the old images `yarn docker:down`
* Rebuild your docker image: `yarn docker:build-image`
* launch the containers `yarn docker:up -d`
* get into `wp shell` by running `yarn docker:wp shell`
* Make sure you see some color output, and your favorite commands still work
* Try to run `wp shell` with build-in REPL: `yarn docker:wp shell --basic`

You should see something like this: 
<img width="1014" alt="1  docker 2019-06-10 18-18-17" src="https://user-images.githubusercontent.com/5654161/59205790-459c0780-8bac-11e9-81b2-d5d54936f829.png">

#### Proposed changelog entry for your changes:

* N/A
